Automattic/wp-calypso
# Dot Pager: Add touch controls.

#### Changes proposed in this Pull Request

* Currently when you visit /home on the mobile phone you can't slide between the different slides using touch gestures. 
* This PR adds touch gestures on mobile devices for the dot-pager component.

### Screenshot

https://user-images.githubusercontent.com/115071/143539894-de7518a2-3695-48fa-b44f-89261a5fd78e.mp4

#### Testing instructions
* Navigate to /home on desktop notice that all the different dot pager components still work as expected. 
* Use the mouse you are able to swipe to the different slides. 
* On a mobile device or simulator visit /devdocs/design/dot-pager notice that the example component works as expected on mobile you can swipe the pages.
